### PR TITLE
Register Subdomain: joabutt

### PR DESCRIPTION
**Pull request by Joabutt**
Added the domain: joabutt

### CODE REVIEW

File: domains/eis.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+ 
+    {
+      "owner": {
+        "username": "Joabutt",
+        "email": "leejoab75@gmail.com"
+      },
+      "record": {
+        "CNAME": ""https://e-4w8.pages.dev""
```
Comment (WilliamDavidHarrison) [inline domains/eis.json:8]:
```suggestion
        "CNAME": "https://e-4w8.pages.dev"
```

File: domains/joabutt.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+ 
+    {
+      "owner": {
+        "username": "Joabutt",
+        "email": "leejoab75@gmail.com"
+      },
+      "record": {
+        "CNAME": ""joabuttisadev.pages.dev""
```
Comment (WilliamDavidHarrison) [inline domains/joabutt.json:8]:
```suggestion
        "CNAME": "joabuttisadev.pages.dev"
```

File: domains/atom [hunk 3]
```diff
@@ -0,0 +1 @@
+
```
Comment (WilliamDavidHarrison) [inline domains/atom:1]: Please rename this file to `atom.json` and add content to it.

**Review by wdhdev — CHANGES_REQUESTED** [review]

File: domains/eis.json [hunk 4]
```diff
@@ -5,7 +5,7 @@
         "email": "leejoab75@gmail.com"
       },
       "record": {
-        "CNAME": ""https://e-4w8.pages.dev""
+        "CNAME": "https://e-4w8.pages.dev"
```
Comment (WilliamDavidHarrison) [inline domains/eis.json:8]: Please add content to your website.

**Review by wdhdev — CHANGES_REQUESTED** [review]

**Comment by Joabutt** [comment]
I have updated. eis.json is not supposed to be there.

**Pull request closed without merging**